FRINXio/frinx-frontend
# FR-232 add stream

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FR-232` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


HOW TO TEST:

test with https://github.com/FRINXio/frinx-inventory-server/pull/434

you should be able to create new stream via ui
